packages/flame_isolate/README.md

```diff
@@ -0,0 +1,40 @@
+The power of [integral_isolates](https://pub.dev/packages/integral_isolates) in your
+[Flame](https://pub.dev/packages/flame) game.
+
+## Usage
```

Comment:
Two empty lines before a new section
(We don't have a markdown linter running atm...)